Development

Abner21 opened the pull request:
Submit for code review

Aaron-RN committed:
Start

Aaron-RN committed:
Page Layout

Abner21 committed:
until the third part

Aaron-RN committed:
Positioning Adjustments

Fixed editors picks sizing
Fixed inflation theory positioning

Aaron-RN committed:
Merge pull request #1 from Aaron-RN/secondHalf

Second half

Aaron-RN committed:
Various changes

Abner21 committed:
videoAdded

Aaron-RN committed:
Layout changes

Various layout changes
addition of video and satelite image sections

Aaron-RN committed:
Added Bottom Links

Footer was added

Aaron-RN committed:
Merge pull request #2 from Aaron-RN/textVideoBottomBar

Text video bottom bar

Aaron-RN committed:
Update README.md

Added Link to Github Page and authors

Abner21 committed:
last changes

Abner21 committed:
Merge pull request #3 from Aaron-RN/footerbar

last changes

Aaron-RN committed:
Minor visual changes

samgaco commented:
Hi @Abner21 and @Aaron-RN :vulcan_salute: 

The project looks amazing, you've done a really good job :clap: . However, there are some points you should address before proceeding; 

* The NYT logo in the top left corner of the footer should be a link.
* **As a suggestion;**  Try to make the readme file more appealing, it's the first thing that someone will see if he enters to your project. You can inspire yourselves with [this template.](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2
)

* It seems like there is a problem with the sizing of the images in the right editor's picks section;

> 
![editors](https://user-images.githubusercontent.com/34820784/61414748-0b323280-a8ef-11e9-8be4-c047018f5461.jpg)

* All images and videos should be enclosed by a `<figure>` tag with a `<figcaption> `for the description. You can find more information about it here: https://www.w3schools.com/tags/tag_figcaption.asp

* The `<figcaption>` and `<figure>` tags should be also used for the left side of additional links. Following this structure:

```
<figure>
  <img src="image.jpg" alt="img">
  <figcaption>Title of the article.</figcaption>
</figure>
```

* Any section including paragraphs in your project should be enclosed within the` <section> `tag. In the case of the "Editor's picks" and the "Sign up" button they should also include the `<aside>` tag. Learn more about it here: https://www.w3schools.com/html/html5_semantic_elements.asp


Please :pray: work on these points and submit again for a code review when you are done. You truly are doing a great job, hope you keep it up :muscle:

Aaron-RN committed:
Reviewer Changes

Added <aside> Tags to Side page elements: Sign up for Science Times  boxand Editors Pick box

Started to add <figure> and <figcaption> tags to image elements

Added anchor to New York Times Logo  in the Footer

Cleans up the spacing in Editors Picks side element in the <aside> tag

Aaron-RN committed:
Minor Adjustments Editors Pick

Margin Adjustments to Editors Pick Box in the <aside> tag

Aaron-RN committed:
Update README.md

Made Readme file more interesting

Aaron-RN committed:
Update README.md

Abner21 committed:
last changes

Abner21 committed:
Merge pull request #5 from Aaron-RN/myBranch

last changes

Aaron-RN committed:
Class naming Error

Fixed Class naming error for the position absolute number "615" placed within the comment buble icon near top of page

Aaron-RN committed:
Reorganized Figcaption tags

Reordered <figcaption> tags to encase only the image's elements related to its caption/descriptive text

Ispirett commented:
Really great job with these changes you were asked to make folks :1st_place_medal: :100: . This site looks beautiful. There are some minor issues to be addressed.
- [ ] Element figcaption not allowed as child of element. This is how figure and figcaption works 
` <figure> <img>  <figcaption>text here</figcaption> </figure>`

When finished with these simple errors submit for another review. Keep up the exceptional work folks!!!

Also important note for feature HTML projects use a feature branch instead of development you can give that branch descriptive name that describes the feature you are implementing.

Aaron-RN committed:
Update index.html

Abner21 committed:
Last Request changes to figure and figcaption

Aaron-RN committed:
Merge branch 'Development' of https://github.com/Aaron-RN/Positioning-and-Floating-Elements into Development

Aaron-RN committed:
Minor Element Alignment changes

Aaron-RN committed:
removed outlines

bolah2009 reviewed (approved):
# Congratulations Team! :tada:

You've done an awesome Job :muscle:

The [live page](https://raw.githack.com/Aaron-RN/Positioning-and-Floating-Elements/Development/index.html) looks good to me :+1: 

Good luck with your next project
Keep it up :star2:

Cheers
Bola
